biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -145,6 +145,17 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	/**
+	 * @return the java release number for this EE
+	 */
+	public int getRelease() {
+		int major = getCapabilityVersion().getMajor();
+		if (major > 8) {
```

Comment:
This is wrong. javac defines --release values for 6, 7, and 8.

Reply:
fixed